src/pages/index.js

```diff
@@ -40,62 +45,185 @@ const section = new Section(
   },
   ".cards__list"
 );
-section.renderItems();
 
-const popupEditForm = new PopupWithForm(
+api
+  .getInitialCards()
+  .then((cards) => {
+    section.setItems(cards);
+    section.renderItems();
+  })
+  .catch(console.error);
+
+/* -------------------------------------------------------------------------- */
+/*                                  Popups                                    */
+/* -------------------------------------------------------------------------- */
+const editProfilePopup = new PopupWithForm(
   "#profile-edit-modal",
   handleProfileEditSubmit
 );
-popupEditForm.setEventListeners();
+editProfilePopup.setEventListeners();
+
+const editAvatarPopup = new PopupWithForm(
+  "#edit-avatar-modal",
+  handleAvatarSubmit
+);
+editAvatarPopup.setEventListeners();
 
-const popupAddForm = new PopupWithForm(
+const addCardPopup = new PopupWithForm(
   "#profile-add-modal",
   handleAddCardSubmit
 );
-popupAddForm.setEventListeners();
+addCardPopup.setEventListeners();
 
 const popupImage = new PopupWithImage("#modal__image");
-popupImage.setEventListiners();
+popupImage.setEventListeners();
 
 const userInfo = new UserInfo({
   titleSelector: ".profile__title",
   descriptionSelector: ".profile__description",
+  avatar: ".profile__image",
 });
 
 // FUNCTIONS/////
 function createCard(cardData) {
-  const card = new Card(cardData, "#card-template", handleImageClick);
-  return card.getCardElement();
+  const card = new Card(
+    cardData,
+    "#card-template",
+    handleImageClick,
+    handleLikeClicks,
+    handleDeleteSubmit
+  );
+  return card.getView();
 }
 
 function renderCard(cardData) {
   const cardElement = createCard(cardData);
   section.addItem(cardElement);
 }
 
-function handleImageClick(name, link) {
-  popupImage.open(name, link);
+function handleImageClick(cardData) {
+  popupImage.open(cardData);
 }
 
-function handleProfileEditSubmit(data) {
-  userInfo.setUserInfo(data);
-  popupEditForm.close();
+function handleAvatarSubmit(url) {
+  editAvatarPopup.renderLoading(true);
+  api
+    .updateAvatar(url)
+    .then((res) => {
+      userInfo.updateAvatar(res);
+      avatarForm.reset();
+      avatarFormValidator.resetValidation();
```

Comment:
`.resetValidation();` is not needed when you submit the card and avatar forms. You have `disableButton` for that